server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -66,8 +61,9 @@ export class DocumentStorage implements IDocumentStorage {
         const tenant = await this.tenantManager.getTenant(tenantId);
         const gitManager = tenant.gitManager;
 
-        const blobsShaCache = new Set<string>();
-        const handle = await writeSummaryTree(gitManager, summary, blobsShaCache, undefined);
+        const blobsShaCache = new Map<string, string>();
```

Comment:
Curios. Why did you convert the set to a map?

Reply:
The r11s-driver's DocumentStorageService is using a Map (although it only needs a Set) so it seemed like the lowest friction option to not alter the internal logic from the driver's perspective